include/swift/AST/DiagnosticsParse.def

```diff
@@ -68,6 +68,9 @@ ERROR(extra_tokens_conditional_compilation_directive,none,
       "extra tokens following conditional compilation directive", ())
 ERROR(unexpected_rbrace_in_conditional_compilation_block,none,
       "unexpected '}' in conditional compilation block", ())
+ERROR(unexpected_if_following_else_compilation_directive,none,
+      "unexpected 'if' keyword following '#else' conditional compilation directive; did you mean '#elseif?'",
+      ())
```

Comment:
Could you break this into more lines?

```macro
ERROR(unexpected_if_following_else_compilation_directive,none,
      "unexpected 'if' keyword following '#else' conditional compilation directive";
      "did you mean '#elseif?'", ())
```

Reply:
Semicolon should be inside the string, and make sure you include the space. But yes, this is generally a good idea.

Reply:
fixed